retrofit-mock/src/main/java/retrofit2/mock/BehaviorCall.java

```diff
@@ -89,6 +89,7 @@ boolean delaySleep() {
               }
 
               @Override
+              @SuppressWarnings("required.method.not.called") //Resource Leak : behavior.createErrorResponse() was casted as a Response<T> and it will become a error Response, therefore it will hold a responseBody that must be closed.
```

Comment:
The comment here seems to contradict the `//noinspection` comment on line 102, which specifically says that an error response has no body (and, presumably, therefore doesn't need to be closed). Given that comment, I suspect this is a false positive.